lib/src/Uncertainty/Algorithm/Sensitivity/SobolIndicesAlgorithmImplementation.cxx

```diff
@@ -186,7 +139,7 @@ Point SobolIndicesAlgorithmImplementation::getFirstOrderIndices(const UnsignedIn
     throw InvalidArgumentException(HERE) << "In SobolIndicesAlgorithmImplementation::getTotalOrderIndices, marginalIndex should be in [0," << outputDimension - 1;
   // return value
   const Point firstOrderSensitivity(varianceI_[marginalIndex] / referenceVariance_[marginalIndex]);
-  for (UnsignedInteger p = 0; p < inputDesign_.getDimension(); ++p)
+  for (UnsignedInteger p = 0; p < inputDescription_.getSize(); ++p)
```

Comment:
It looks like inputDesign_.getDimension() has a problem if you refuse to use it. What is the underlying problem?

Reply:
I removed inputDesign_ attribute: it is unused in the computation!